includes/classes/class-omise-card-image.php

```diff
@@ -12,7 +12,7 @@ class Omise_Card_Image {
 		 */
 		private static function get_image( $file, $alternate_text ) {
 			$url = WC_HTTPS::force_https_url( WC()->plugin_url() . '/assets/images/icons/credit-cards/' );
-			return "<img src='$url/$file' width='38px' alt='$alternate_text' />";
+			return "<img src='$url/$file' style='width: 38px;' alt='$alternate_text' />";
```

Comment:
@nimid ✋ have one question here, do you have any idea/plan to support in case people want to overwrite this rule (width 38px) by themselves?
For example, they want to control the sizing of the icons on desktop/mobile screens to have difference width.